### PR TITLE
Update for the Winget command

### PR DESCRIPTION
**Pull request by fskelly**
Seems like the command for Winget has changed slightly

### CODE REVIEW

**Comment by PRMerger20** [comment]
@fskelly : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by franky920920** [comment]
Linked issue is https://github.com/microsoft/PowerToys/issues/13579

**Comment by ktoliver** [comment]
#label:"aq-pr-triaged"

File: hub/powertoys/install.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ To install PowerToys using the Windows Package Manager (WinGet) preview:
 2. Run the following command from the command line / PowerShell:
 
 ```powershell
-WinGet install powertoys
+winget install Microsoft.PowerToys --source winget
```
Comment (dedavis6797) [inline hub/powertoys/install.md:52]:
```suggestion
winget install powertoys -s winget
```

Comment (dedavis6797) [reply hub/powertoys/install.md:52]: More concise. Thanks for creating this PR!

**Pull request merged**